### PR TITLE
Add a write_config command

### PR DESCRIPTION
**Pull request by julienw**

### CODE REVIEW

**Comment by julienw** [comment]
This needs https://github.com/fxbox/openzwave-rust/pull/28

File: src/main.rs [hunk 1]
```diff
@@ -105,6 +105,14 @@ fn main() {
                     println!("Must specify a numeric home_id (in base 16)");
                 }
             }
+            "write_config"          => {
+                let state = ozw.get_state();
+                let controllers = state.get_controllers();
+                for (controller, _) in controllers {
+                    controller.write_config();
+                }
```
Comment (julienw) [inline src/main.rs]: wondering if we could add a write_configs in the ozw object that would call write_config on all controllers automatically ? what do you think ?

Comment (julienw) [inline src/main.rs]: ok, I did it after all :)

**Comment by julienw** [comment]
So this needs https://github.com/fxbox/openzwave-stateful-rust/pull/16 too

**Comment by dhylands** [comment]
r+

**Pull request merged**